packages/shared-ux/page/kibana_no_data/impl/src/services.tsx

```diff
@@ -47,10 +47,11 @@ export const KibanaNoDataPageKibanaProvider: FC<KibanaNoDataPageKibanaDependenci
   children,
   ...dependencies
 }) => {
-  const { dataViews } = dependencies;
+  const { dataViews, customBranding } = dependencies;
   const value: Services = {
     hasESData: dataViews.hasData.hasESData,
     hasUserDataView: dataViews.hasData.hasUserDataView,
+    hasCustomBranding: customBranding.hasCustomBranding,
```

Comment:
I was thinking of implementing it this way versus adding onto coreStart in the types package (having a customBranding property alongside docLinks in `index.d.ts`) because customBranding is its own service independent of coreStart. Am I thinking of this correctly?

Reply:
`customBranding` is a service exposed from `coreStart`, it has already been added.

Reply:
Ok thanks that cleared it up 😅  I think [cc4d8e0](https://github.com/elastic/kibana/pull/149253/commits/cc4d8e04ded8616a3853eda86b11e72f2ad034e1) makes sense now